### ISSUE TITLE
[Feature Request] Drag and drop representation could be a stack of files

### ISSUE THREAD

**Issue by reece146** [elementary/files#1000]
When drag and dropping multiple files simultaneously from one Files window to the next, the representation of the files attached to the mouse pointer is a single file. The first file is shown on the pointer. 

It would be a nice touch if the files attached to the mouse pointer was a "stack" of files like seen in other OS.

Not sure if this reads as intended (diction). Please ask for clarification if required.

**Comment by jeremypw** [comment]
Thanks for the report -  I understand your point. We would need to decide on a suitable icon to use.

**Comment by janez33** [comment]
It also gets stuck when you first time drag & droping files, it's a serious issue I guess. Can you guys confirm it or should I open new ticket?

**Comment by jeremypw** [comment]
@janez33 Yes, you should open a new issue if this has not been raised before.  I have not experienced this.  Make sure you give full details as to which version you are running and how to reproduce the bug, thanks.

**Comment by jeremypw** [comment]
@danirabbit Is there a suitable icon that could be used?

**Comment by danirabbit** [comment]
@jeremypw i don't think so. I'll assign to UX and put it on the OS 9 project board so we can remember to make one
